iwyu_test_util.py

```diff
@@ -203,6 +204,18 @@ def _StripCommentFromLine(line):
     return line
 
 
+def _IsCommentLine(line):
```

Comment:
I think this can be spelled out inline, no need to have an abstraction for something as simple as checking `//`